namui/src/namui/render/text_input/mod.rs

```diff
@@ -144,7 +144,19 @@ impl TextInput {
         render([
             namui::rect(RectParam {
                 rect: props.rect,
-                style: props.style.rect,
+                style: RectStyle {
+                    stroke: if props.style.rect.stroke.is_some() || props.style.rect.fill.is_some()
+                    {
+                        props.style.rect.stroke
```

Comment:
Is `props.style.rect.fill.is_some()` related to `props.style.rect.stroke`?

Reply:
Yes. If fill is some, they will render rect path using fill paint, so it will be caught during calculating bounding box